template_engine.rb

```diff
@@ -0,0 +1,95 @@
+require "active_support/all"
+
+run_ruby_script("bin/rails plugin new admin --mountable")
+
+gsub_file "admin/admin.gemspec", "TODO: ", ""
+gsub_file "admin/admin.gemspec", "TODO", ""
+
+inject_into_file "admin/admin.gemspec", before: /^end/ do
+  <<-END.strip_heredoc.indent(2)
+    s.add_dependency "godmin", "> 0.12"
```

Comment:
I guess we need a specified version here, because it's a gemspec.. Not sure what is appropriate.